### PR TITLE
feature(ui-ux): dex home screen header v2

### PR DESCRIPTION
**Pull request by fullstackninja864**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:
This Pr contains v2 screens for
- Home screen header
- Category tab
- Total Value Locked
Note: There is misalignment in dex landing screen related to component position

This will get fixed in #3330 

#### Which issue(s) does this PR fixes?:

Fixes #

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [ ] No console errors on web
- [ ] Tested on Light mode and Dark mode*
- [ ] Your UI implementation visually matched the rendered design*
- [ ] Unit tests*
- [ ] Added e2e tests*
- [ ] Added translations*

### CODE REVIEW

File: mobile-app/app/screens/AppNavigator/screens/Dex/DexNavigator.tsx [hunk 1]
```diff
@@ -98,15 +112,42 @@ export function DexNavigator (): JSX.Element {
         component={DexScreen}
         name='DexScreen'
         options={{
+          ...screenOptions,
+          headerLeft: undefined,
+          headerLeftContainerStyle: null,
+          headerTitleAlign: 'left',
+          headerBackTitleVisible: false,
+          headerTitleContainerStyle: tailwind('mt-4 ml-5'),
+          headerRightContainerStyle: [screenOptions.headerRightContainerStyle, tailwind('mt-5 justify-start', { 'pr-3': Platform.OS !== 'ios' })],
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Dex/DexNavigator.tsx:121]: does this works? works on my both devices
```suggestion
          headerRightContainerStyle: [screenOptions.headerRightContainerStyle, tailwind('mt-5 justify-start', { 'pr-5': Platform.OS !== 'ios' })],
```

File: mobile-app/app/screens/AppNavigator/screens/Dex/DexNavigator.tsx [hunk 2]
```diff
@@ -98,15 +112,42 @@ export function DexNavigator (): JSX.Element {
         component={DexScreen}
         name='DexScreen'
         options={{
+          ...screenOptions,
+          headerLeft: undefined,
+          headerLeftContainerStyle: null,
+          headerTitleAlign: 'left',
+          headerBackTitleVisible: false,
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Dex/DexNavigator.tsx:119]: repeated in `screenOptions`
```suggestion

```

File: mobile-app/app/screens/AppNavigator/screens/Dex/DexNavigator.tsx [hunk 3]
```diff
@@ -98,15 +112,42 @@ export function DexNavigator (): JSX.Element {
...
         }}
       />
 
+      <DexStack.Screen
+        component={NetworkSelectionScreen}
+        name='NetworkSelectionScreen'
+        options={{
+          ...screenOptions,
+          headerTitle: translate('screens/NetworkSelectionScreen', 'Network'),
+          headerBackTitleVisible: false,
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Dex/DexNavigator.tsx:146]: repeated in `screenOptions`
```suggestion

```

File: mobile-app/app/screens/AppNavigator/screens/Dex/DexNavigator.tsx (hunk 1 above)
Comment (kyleleow) [reply mobile-app/app/screens/AppNavigator/screens/Dex/DexNavigator.tsx:121]: are we not including this change? without it the header network is mis-aligned on my android @fullstackninja864

Comment (fullstackninja864) [reply mobile-app/app/screens/AppNavigator/screens/Dex/DexNavigator.tsx]: Updated in recent commit

**Review by kyleleow — APPROVED** [review]

**Pull request merged**